inspirehep/modules/workflows/static/js/inspire_workflows_ui/templates/details.html

```diff
@@ -431,7 +431,7 @@
             <button class="btn btn-default"
                     ng-if="vm.record._workflow.status !== 'HALTED'"
                     ng-click="Utils.resumeWorkflow()">
-              <i class="fa fa-play" aria-hidden="true"></i> Resume
+              <i class="fa fa-play" aria-hidden="true"></i> Skip to next step
```

Comment:
I don't think that this should be addressed in this pull request, but is a very good question.

About it showing, probably we should filter out also the 'running' and 'inital' states... though I've found that there's nowhere a complete statement of what those states are or should be, we (or me, whomever) should first figure out what states are there.

About it's usefulness, well, no idea what they had in mind when they put that button. Maybe if refexctract failed, and you don't care and just want to keep going or something. I don't find many other cases where it might be not just useful, but even safe.